exercises/string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,13 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  var truth = false;
```

Comment:
If you start this as `true`, then you can revise this to match what I described for your `isUpperCased` function